biojava-core/src/main/java/demo/UncompressFile.java

```diff
@@ -0,0 +1,42 @@
+package demo;
```

Comment:
This new file (and others in this pull request) need the license header.  For example
https://github.com/biojava/biojava/blob/master/biojava-aa-prop/src/main/java/demo/BioJavaAADemo.java#L1